test/integration/features/basic.feature

```diff
@@ -10,6 +10,42 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then body of HTTP request to "OpenShift" at "/healthz" contains "ok"
```

Comment:
Very nice, I like this a lot. I also like how this handles the dynamic part. So, if I get this right

    Then body of HTTP request to "OpenShift" at "/healthz" contains "ok"

Will check the actual OpenShift URL + whatever is specified under "at", but I could also do:

    Then body of HTTP request to "http://foo.bar.com" contains "ok"

Is this correct.